tools/c7n_azure/c7n_azure/container_host/host.py

```diff
@@ -426,32 +427,52 @@ def build_options(output_dir=None, log_group=None, metrics=None):
 
         return Azure().initialize(config)
 
+    @staticmethod
+    def get_scheduler_config():
+        return {
+            'apscheduler.jobstores.default': {
+                'type': 'memory'
+            },
+            'apscheduler.executors.default': {
+                'class': 'apscheduler.executors.pool:ProcessPoolExecutor',
+                'max_workers': '4'
```

Comment:
We could add a work item if you don't want to do this now.

We could take both threadpool and workers  in as a config parameter and and default to 4.